content/reference-architectures/cold-start.md

```diff
@@ -0,0 +1,598 @@
+make init
+
+# Build docker image
+make docker/build
+
+# Install the wrapper shell
+make install
+
+# Run the shell
+testing.cloudposse.co
+
+# Login to AWS as the admin user with your MFA device
+assume-role
+```
+{{% /dialog %}}
+
+You should see the `Docker` image built, `geodesic` shell started, and after you run the `assume-role` command, you will be logged in to the `testing` account:
+
+```
+Enter passphrase for /localhost/.ssh/id_rsa:
+Identity added: /localhost/.ssh/id_rsa (/localhost/.ssh/id_rsa)
+-> Run 'assume-role' to login to AWS
+ ⧉  testing.cloudposse.co
+❌   (none) ~ ➤  assume-role
+Enter passphrase to unlock /conf/.awsvault/keys/:
+* Assumed role arn:aws:iam::126450723953:role/OrganizationAccountAccessRole
+```
+
+
+## Provision `tfstate-backend` Project for `testing`
+
+Execute this sequence of steps in the `testing` geodesic session:
+
+{{% dialog type="code-block" icon="fa fa-code" title="Provision `tfstate-backend` Project for `testing`" %}}
+```
+cd tfstate-backend
+
+Comment out the `backend          "s3"             {}` section in `tfstate-backend/main.tf`
+
+Comment out the `assume_role` section in `tfstate-backend/main.tf`
+
+Run `init-terraform`
+
+Run `terraform plan` and then `terraform apply`
+
+Re-enable `backend          "s3"             {}` section in `tfstate-backend/main.tf`
+
+Re-run `init-terraform`
+
+Re-run `terraform apply`, answer `yes` when asked to import state
+
+Re-enable the `assume_role` section in `tfstate-backend/main.tf`
+```
+{{% /dialog %}}
+
+{{% dialog type="info" icon="fa-info-circle" title="Note" %}}
+You could use the following commands to comment out and then uncomment the `backend` section:
+
+```shell
+sed -i "s/ backend / #backend /" main.tf
+sed -i "s/ #backend / backend /" main.tf
+```
+{{% /dialog %}}
+
+Now we have the S3 bucket and DynamoDB table provisioned, and Terraform state stored in the bucket itself.
+
+
+## Provision `account-dns` Project in `testing` to Create `testing` DNS Zone
+
+In `testing` `geodesic` shell, execute the following commands:
+
+{{% dialog type="code-block" icon="fa fa-code" title="Provision `account-dns` Project in `testing` to Create `testing` DNS Zone" %}}
+```shell
+cd account-dns
+init-terraform
+terraform apply
+```
+{{% /dialog %}}
+
+You should see Terraform output similar to this:
+
+{{% dialog type="code-block" icon="fa fa-code" title="Terraform output" %}}
+```
+name_servers = [
+    ns-1416.awsdns-49.org,
+    ns-1794.awsdns-32.co.uk,
+    ns-312.awsdns-39.com,
+    ns-619.awsdns-13.net
+]
+zone_id = Z3SO0TKDDQ0RGG
+```
+{{% /dialog %}}
+
+Take the Name Servers from the output and update them in the `root` `Dockerfile` (variable `TF_VAR_testing_name_servers`).
+
+
+## Rebuild And Restart `root` `geodesic` Shell
+
+Rebuild and restart the `root` `geodesic` shell by executing the following commands:
+
+{{% dialog type="code-block" icon="fa fa-code" title="Rebuild And Restart `root` `geodesic` Shell" %}}
+```shell
+exit
```

Comment:
I think we should say instead:

Exit the shell, then run these commands.

```
make docker/build`
....
```

The `exit` part is immaterial to the action. It's kind of like saying "Move the mouse over the submit button. Then click the submit button", rather than just saying "click the submit button."